netlify.toml

```diff
@@ -1,3 +1,6 @@
 [build]
   command = "yarn build:docs"
   publish = "docs/dist"
+
+[build.environment]
```

Comment:
`@storybook/addon-postcss` has an engines key in the `package.json` file that was causing the build to fail for the docs unless I updated this.